### DISCUSSION TITLE
Code executed when yarn start is run

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#40568] [Help] [answered]
I'm new to Next.js.
I'd like to see the code executed when I run the yarn start script. Can you tell me where the github repository is?

**Comment by gaston-flores** [comment, marked as answer]
You can find [all the commands here](https://github.com/vercel/next.js/tree/canary/packages/next/cli), and this is [next start](https://github.com/vercel/next.js/blob/canary/packages/next/cli/next-start.ts)

**Reply by ghost** [reply]
@gaston-flores   Thank you  : )
